modules/serverless-installing-cli-linux-ibm-power-rpm.adoc

```diff
@@ -0,0 +1,43 @@
+[id="installing-cli-linux-ibm-power-rpm_{context}"]
```

Comment:
@abrennan89  I've elaborated "Enter the command" for this module, can you please check this once then i'll make changes to other module(for Z)